karma.conf.js

```diff
@@ -0,0 +1,88 @@
+/* eslint no-param-reassign:0, import/no-extraneous-dependencies:0 */
+const path = require('path');
+const webpack = require('webpack');
+
+module.exports = (config) => {
+  config.set({
+    basePath: '',
+
+    frameworks: ['mocha', 'sinon', 'chai'],
+
+    files: [
+      'test/_helpers/*.js',
+      'test/utils/*.js',
+      'test/components/*.jsx',
```

Comment:
do we have to specify the file extensions here? it seems more robust to just have "*"